src/globals.c

```diff
@@ -411,6 +411,16 @@ bool set_default_urls()
 	return true;
 }
 
+/* Sets the repo_name_3rd_party global variable */
+void set_repo_name_3rd_party(char *repo_name)
```

Comment:
:+1:  This is a good idea, maybe I can also set this in repo_manage and use it to determine path where repos will be created.